### PR TITLE
PR for configuration with 2 strings and 3 phases

### PR DESCRIPTION
**Pull request by maubreville**
First of all: Thanks for this magnificent library! It works very nicely, after a little change I implemented. Apparently the configuration here is 3 phases but 2 strings, which was not yet covered by the current version of the library.

### CODE REVIEW

File: kostalpiko/const.py [hunk 1]
```diff
@@ -22,6 +22,17 @@
     "status": 11 
 }
 
+DOUBLE_STRING_THREE_PHASES_INDICES = {
+    **BASE_INDICES,
+    "string2_voltage": 7,
+    "l2_voltage": 8,
+    "string2_current": 9,
+    "l2_power": 10,
+    "l3_voltage": 8,
```
Comment (rcasula) [inline kostalpiko/const.py]: why `l3_voltage ` and `l2_voltage` have the same index?

File: kostalpiko/const.py [hunk 2]
```diff
@@ -22,6 +22,17 @@
...
+    "l3_power": 10,
```
Comment (rcasula) [inline kostalpiko/const.py]: same here with `l2_power`

**Review by rcasula — CHANGES_REQUESTED** [review]

**Review by rcasula — APPROVED** [review]

**Comment by maubreville** [comment]
Thanks! 👍

**Comment by rcasula** [comment]
Thank you!

**Pull request merged**